GreenGarbageApp/app/src/main/res/layout/fragment_review.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingTop="180dp"
+    tools:context="com.example.greengarbageapp.activities.MainActivity">
+
+    <TextView
+
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="8dp"
+
+        android:fontFamily="@font/acme"
+        android:textColor="@color/green_dark"
+        android:gravity="center"
+        android:text="@string/help_us_improve_the_app"
+        android:textSize="35sp"/>
+    <Space
+        android:layout_width="match_parent"
+        android:layout_height="50dp" />
+
+    <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="80dp"
+        android:layout_gravity="center_vertical|center_horizontal"
+        android:src="@drawable/smile" />
+    <Space
+        android:layout_width="match_parent"
+        android:layout_height="50dp" />
+    <RatingBar
```

Comment:
i would suggest to change the color of the stars (when clicked) to be in a color that better suits the add. 
Just add:
android:progressTint="@color/..."